clab/config/utils.go

```diff
@@ -13,16 +14,16 @@ import (
 )
 
 const (
-	vkNodes    = "nodes" // reserved, used for all nodes
-	vkNodeName = "node"  // reserved, used for the node's ShortName
-	vkLinks    = "links" // reserved, used for all link in a node
-	vkRole     = "role"  // optional, will default to the node's Kind. Used to select the template
-	vkFarEnd   = "far"   // reserved, used for far-end link & node info
-
-	vkSystemIP = "systemip" // optional, system IP if present could be used to calc link IPs
-	vkLinkIP   = "ip"       // optional, link IP
-	vkLinkName = "name"     // optional, from ShortNames
-	vkLinkNr   = "linknr"   // optional, link number in case you have multiple, used to calculate the name
+	vkNodeName = "node"       // reserved, used for the node's ShortName
+	vkNodes    = "clab_nodes" // reserved, used for all nodes
+	vkLinks    = "clab_links" // reserved, used for all link in a node
+	vkFarEnd   = "clab_far"   // reserved, used for far-end link & node info
+	vkRole     = "clab_role"  // optional, will default to the node's Kind. Used to select the template
+
+	vkSystemIP = "system_ip" // optional, system IP if present could be used to calc link IPs
```

Comment:
why not `clab_system_ip` to indicate this will potentially be used by clab to do stuff?

Reply:
Ok, will add `clab_`, though intention was to use clab_ for topology file related items and without for more logical related items items

Reply:
A bit of a slippery slope, since `port` will have to be renamed to `clab_port` in the supplied templates once we auto-populate them